### DISCUSSION TITLE
Bindings for datetimes in automation

### DISCUSSION THREAD

**Discussion by floredelasteyrie** [Budibase/budibase#4656] [Help] [answered]
Hello, 

Is it possible to use bindings for datetimes one way or another in automations ?
I'm trying to automatically create a row on some trigger, but as my table has "created_at" / "updated_at" columns, I need to be able to give it the current datetime (for example with `JS Date.now()`). How would you do that ?
Currently I only manage to manually choose a date when creating the automation, which doesn't really work ...

<img width="497" alt="Screenshot 2022-02-23 at 11 29 41" src="https://user-images.githubusercontent.com/41300908/155302511-74e1c3e0-2995-44db-8b69-c3572c8be0c3.png">

**Comment by PClmnt** [comment, marked as answer]
Hi @floredelasteyrie, I've got a PR open for this: https://github.com/Budibase/budibase/pull/4390 Should be available in the next release.

**Reply by floredelasteyrie** [reply]
Great, I didn't see that, thanks for the quick answer and future feature :)
